pkg/v5/configmap/configmap.go

```diff
@@ -58,9 +58,9 @@ func containsConfigMap(list []*corev1.ConfigMap, item *corev1.ConfigMap) bool {
 	return false
 }
 
-func getConfigMapByName(list []*corev1.ConfigMap, name string) (*corev1.ConfigMap, error) {
+func getConfigMapByNameAndNamespace(list []*corev1.ConfigMap, name, namespace string) (*corev1.ConfigMap, error) {
```

Comment:
Since the configmaps can be created in multiple namespaces we need to check both.